client/my-sites/people/edit-team-member-form/edit-user-form.jsx

```diff
@@ -273,9 +273,11 @@ export default localize(
 	connect(
 		( state, { siteId, user } ) => {
 			const externalContributors = ( siteId && requestExternalContributors( siteId ).data ) || [];
+			const userId = user?.linked_user_ID || user?.ID;
```

Comment:
The `user` prop is guaranteed to be a valid object, and there are places in this component that destructure it without checking. If you don't mind a little drive-by cleanup, all optional chains when accessing `user` can be removed.

Reply:
Good point, [just noticed it](https://github.com/Automattic/wp-calypso/blob/d64f25956d54275c213f948b3e06180267d73482/client/my-sites/people/edit-team-member-form/index.jsx#L67). 

Happy to clean this up here, this PR is very small anyway.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/52238/commits/99f8fc8028345f065b3091d6f92dc9c6fa36c232.